retriever/lib/engine_tools.py

```diff
@@ -98,7 +98,7 @@ def reset_retriever(scope="all", ask_permission=True):
     """Remove stored information on scripts and data."""
     warning_messages = {
         'all': "\nThis will remove existing scripts and cached data."
-               "\nSpecifically it will remove the scripts and raw_data folders"
+               "\nSpecifically it will remove the scripts and raw_data folders "
                "in {}\nDo you want to proceed? (y/N)\n",
```

Comment:
Remove the space created

Reply:
There was a small typo. This extra space is added to add a space between the words `folders` and `in` otherwise it resulted into `foldersin` when `retriever reset all` was done.

After adding a space:
![Screenshot from 2019-03-14 08-26-37](https://user-images.githubusercontent.com/21103831/54328243-39e38800-4633-11e9-86e1-2bf10cf075ac.png)